test/fixtures/linter/autofix/ansi/001_long_line/violations.json

```diff
@@ -2,7 +2,7 @@
     "violations":{
         "linting":{
             "L016": [
-                [3, 199]
+                [3, 9]
```

Comment:
Can you explain this change?

Reply:
L016 used to anchor on the final newline at the end of the line. It now anchors on the first non-indent character of the line. That felt a little more consistent.